lib/session/session.go

```diff
@@ -110,6 +110,48 @@ type Session struct {
 	Moderated bool `json:"moderated"`
 }
 
+// FileTransferParams contain parameters for requesting a file transfer
```

Comment:
This is a bit misleading, because it looks like this can be a request or a response based on the fields below.

Reply:
I agree. A comment on the other PR lead me to separate these from requests and responses so it will change (not sure outcome yet, but will update comment regardless)